bot/commands/commands.py

```diff
@@ -12,8 +12,14 @@
 @register('botrole')
 @register('role')
 async def botrole(command, msg, user, channel, *args, **kwargs):
+    botrole = await read_db('guild', 'botrole', kwargs['guild'])
+    try:
+        botrole_name = channel.guild.get_role(botrole).name
+    except:
+        botrole_name = '@everyone'
```

Comment:
should be `everyone` not `@everyone`. Right now all other names will have no @ so it doesn't accidentally ping the role

Reply:
Solved in newest commit